tests/CampiagnsTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Rogue\Services\CampaignService;
+use Illuminate\Support\Facades\Cache;
+use Illuminate\Foundation\Testing\WithoutMiddleware;
+
+class CampiagnsTest extends TestCase
+{
+    /**
+     * A basic test example.
+     *s
```

Comment:
this shouldn't even be here haha something from a different branch i accidentally added 😬